api/v1beta1/flowcollector_types.go

```diff
@@ -461,6 +461,10 @@ const (
 
 // FlowCollectorLoki defines the desired state for FlowCollector's Loki client.
 type FlowCollectorLoki struct {
+	//+kubebuilder:default:=true
+	// enable the processor LokiWrite stage.
```

Comment:
Since this comment is what users will see in the FlowCollector definition, it should be more user-oriented than technical-oriented - and also warn that turning it off is going to disable the console plugin. So maybe something like "enable storing flows to Loki. It is required for the OpenShift Console plugin installation."